MediaBrowser.Controller/Entities/BaseItem.cs

```diff
@@ -2548,14 +2548,24 @@ public bool UpdateRatingToItems(IList<BaseItem> children)
                 StringComparison.OrdinalIgnoreCase);
         }
 
-        public IReadOnlyList<BaseItem> GetThemeSongs()
+        public IReadOnlyList<BaseItem> GetThemeSongs(User user = null)
         {
-            return GetExtras().Where(e => e.ExtraType == Model.Entities.ExtraType.ThemeSong).ToArray();
+            return GetThemeSongs(user, Array.Empty<(ItemSortBy, SortOrder)>());
         }
 
-        public IReadOnlyList<BaseItem> GetThemeVideos()
+        public IReadOnlyList<BaseItem> GetThemeSongs(User user, IEnumerable<(ItemSortBy SortBy, SortOrder SortOrder)> orderBy)
         {
-            return GetExtras().Where(e => e.ExtraType == Model.Entities.ExtraType.ThemeVideo).ToArray();
+            return LibraryManager.Sort(GetExtras().Where(e => e.ExtraType == Model.Entities.ExtraType.ThemeSong), user, orderBy).ToArray();
+        }
+
+        public IReadOnlyList<BaseItem> GetThemeVideos(User user = null)
+        {
+            return GetThemeVideos(user, Array.Empty<(ItemSortBy, SortOrder)>());
+        }
+
+        public IReadOnlyList<BaseItem> GetThemeVideos(User user = null, IEnumerable<(ItemSortBy SortBy, SortOrder SortOrder)> orderBy = null)
```

Comment:
```suggestion
        public IReadOnlyList<BaseItem> GetThemeVideos(User user = null, IEnumerable<(ItemSortBy SortBy, SortOrder SortOrder)> orderBy)
```

Reply:
Nice catch. Actually I'll update the suggestion as the `user` parameter in the two argument overload also shouldn't be defaulted in this case either.

The calling patterns should be
* `GetThemeVideos()` -> 1 argument overload
* `GetThemeVideos(user)` -> 1 argument overload
* `GetThemeVideos(user, orderBy)` -> 2 argument overload

```suggestion
        public IReadOnlyList<BaseItem> GetThemeVideos(User user, IEnumerable<(ItemSortBy SortBy, SortOrder SortOrder)> orderBy)
```